genoverse/modules/EnsEMBL/Web/ImageConfig/Genoverse.pm

```diff
@@ -49,6 +49,7 @@ sub init_genoverse {
   $self->modify_configs([ map "${_}structural_variation", '', 'somatic_' ], { genoverse => { type   => 'StructuralVariation',                  threshold => 5e6   } });
   $self->modify_configs([ 'scalebar', 'ruler', 'draggable', 'info'       ], { genoverse => { remove => 1                                                          } });
   $self->modify_configs([ 'gencode'                                      ], { genoverse => { type   => 'Gene'                                                     } });  
+  $self->modify_configs([ 'mane_select'                                  ], { display => 'off'                                                                      });
```

Comment:
Shouldn't there be a MANE Plus Clinical here as well?

Reply:
No, MANE Plus Clinical is off by default so we don't need to turn it off again here:
https://github.com/Ensembl/ensembl-webcode/pull/873/files